Dockerfile.api

```diff
@@ -1,21 +1,21 @@
+# Use a minimal image for production (e.g., an Alpine-based image with Python)
 FROM python:3.11-slim AS base
 
+# Set the working directory
 ENV COLANDR_APP_DIR=/app
 RUN mkdir -p ${COLANDR_APP_DIR}
 WORKDIR ${COLANDR_APP_DIR}
 
+# Install dependencies (use a requirements.txt)
 RUN apt update \
     && apt install -y g++ git \
     && apt clean \
     && rm -rf /var/lib/apt/lists/* /usr/share/doc /usr/share/man
 
 COPY requirements/ ./requirements/
 RUN python -m pip install --upgrade pip wheel && python -m pip install -r requirements/prod.txt
-RUN python -m textacy download lang_identifier --version 3.0 \
-    && python -m spacy download en_core_web_md \
-    && python -m spacy download es_core_news_md \
-    && python -m spacy download fr_core_news_md
```

Comment:
This isn't installing `textacy`, it's downloading NLP language data that we need for it to operate. I don't think you can remove these lines and have the app's NLP functionality work.